test/node-examples.js

```diff
@@ -46,7 +46,7 @@ async function validateWithWaitingTime(child, folder, waitingTime, threshold) {
   await page.waitFor(waitingTime);
   await page.screenshot({path: 'new.png'});
 
-  const goldImageData = fs.readFileSync(`../test/golden-images/${folder}.png`);
+  const goldImageData = fs.readFileSync(`../../test/render/golden-images/examples/${folder}.png`);
```

Comment:
Can we have an "test/example/golden-images" folder  where we put the golden images for examples?

Reply:
The current golden images are in `test/render/golden-images`, and this PR moves the example golden images into its `examples` subfolder.